### ISSUE TITLE
parcel-plugin-transcrypt broken

### ISSUE THREAD

**Issue by eruvanos** [QQuick/Transcrypt#821]
Using the code from `https://github.com/rtp-book/project` the client build exits with the following error:

```
...
Error: Cannot find module 'parcel-bundler/src/Logger'
...
```

The easy fix is to replace line 2 in the plugins `asset.js` with:
`const logger = require('@parcel/Logger');`

The plugin links to this page, not sure where else I could create an issue for this.

**Comment by JennaSys** [comment]
The `parcel-plugin-transcrypt` library has been broken for some time.  I posted a [how-to](https://dev.to/jennasys/creating-react-applications-with-python-2je1) a while back that discusses a fix that changes line 2 from:
```
const logger = require('parcel-bundler/src/Logger');
```
to:
```
const logger = require('@parcel/logger/src/Logger');
```

That said, being that Parcel v2 is out and v1 has been deprecated, I don't see the v1 plugin ever being fixed at this point. The good news is that there is a [new Parcel plugin available for Transcrypt](https://www.npmjs.com/package/parcel-transformer-transcrypt) that works with Parcel v2. I also did a writeup on [migrating from Parcel v1 to Parcel v2](https://dev.to/jennasys/using-parcel-v2-and-transcrypt-to-create-web-applications-with-python-30ac) using the new plugin. The new plugin works pretty much the same way as the old one, but there a few extra details to pay attention to.

_Disclosure: I'm the person that wrote the book that the code in the repo you are using came from.  I also created the new Parcel plugin :-)_
